payments/lightning-payments.md

```diff
@@ -121,54 +147,88 @@ Covered by potentially BOLT12, still in development...
 - [Announcement on `lightning-dev` mailing list](https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-November/002276.html)
 - [WIP PR on Github](https://github.com/lightningnetwork/lightning-rfc/pull/798)
 
----
 
-### 1. Notes on `key send`
+### Multi-part Payments
+
+Methods for sending multiple LN payments (HTLCs) to make up a single logical payment. The reasoning behind this is to allow for splitting a payment across multiple routes. A person could want to do this for a number of reasons including:
```

Comment:
are multi path payments a node configuration or something that is specified in an invoice?